From 79bf1834b352142c13a43dfbf9c33dba20e019f4 Mon Sep 17 00:00:00 2001
From: Eddie Cai <eddie.cai.linux@gmail.com>
Date: Thu, 23 Feb 2017 09:59:46 +0800
Subject: [PATCH] rockchip: use spl_early_init instead of spl_init

use spl_early_init to avoid malloc_base, limit, ptr not initualized.

Signed-off-by: Eddie Cai <eddie.cai.linux@gmail.com>
---
 arch/arm/mach-rockchip/rk3288-board-spl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-rockchip/rk3288-board-spl.c b/arch/arm/mach-rockchip/rk3288-board-spl.c
index 930939a..02ab5a4 100644
--- a/arch/arm/mach-rockchip/rk3288-board-spl.c
+++ b/arch/arm/mach-rockchip/rk3288-board-spl.c
@@ -184,7 +184,7 @@ void board_init_f(ulong dummy)
 	debug_uart_init();
 #endif
 
-	ret = spl_init();
+	ret = spl_early_init();
 	if (ret) {
 		debug("spl_init() failed: %d\n", ret);
 		hang();
-- 
2.12.0

